packages/squiggle-lang/src/public/SqValue/SqDict.ts

```diff
@@ -30,10 +30,10 @@ export class SqDict {
   }
 
   toString() {
-    return vDict(this._value).toString();
+    return this._value.valueToString();
```

Comment:
Why `valueToString()` and not `toString()`?

I'm not sure where we use this, but `toString()` (which would keep tags) looks more correct to me.

Reply:
Ah, I didn't realize that ``toString`` was there like that, changing, and making ``valueToString()`` protected to help with the confusion.